samigo/samigo-app/src/webapp/jsf/delivery/item/deliverMultipleChoiceMultipleCorrect.jsp

```diff
@@ -50,9 +50,9 @@ should be included in file importing DeliveryMessages
           <h:outputText value=" #{selection.answer.label}" escape="false" />
           <h:outputText value="#{deliveryMessages.dot} " rendered="#{selection.answer.label ne ''}" />
         </span>
-        <h:outputLabel for="samigo-mc-mc" value="#{selection.answer.text}" escape="false">
```

Comment:
I think this solution would introduce an a11y issue when dissociating the checkbox from the label. But I think it could make sense to set the span above as the label.

Reply:
Yeah, good idea.

Reply:
I have one more doubt with this. I think if you don't use `<h:outputLabel>`, the association will not work, as the input id will have the form name as prefix. Maybe the jsf validation might even fail.

Reply:
You're right, blame me.

Let's use a simple CSS rule for this, minimum risk.

Reply:
Yeah, I think CSS is safer and conserves the current behavior.